heureka/ui/src/components/components/ComponentsListController.jsx

```diff
@@ -73,22 +87,30 @@ const ComponentsListController = () => {
 
   return (
     <>
-      <Container py>
-        <ComponentsList components={components} isLoading={isLoading} />
-      </Container>
-      <Stack className="flex justify-end">
-        <Pagination
-          currentPage={currentPage}
-          isFirstPage={currentPage === 1}
-          isLastPage={currentPage === totalPages}
-          onPressNext={onPressNext}
-          onPressPrevious={onPressPrevious}
-          onKeyPress={onKeyPress}
-          onSelectChange={onPaginationChanged}
-          pages={totalPages}
-          variant="input"
-        />
-      </Stack>
+      {!!error ? (
+        <Container py>
```

Comment:
You don't have to render the messages container just when error happens. This is something that the library does for you. Just render <Messages /> always and it will display errors if needed.

Reply:
I tried this, but as there are diferent tabs, without this check, on each tab at least one error message is displayed! therefore I kept it here.